src/metabase/analytics/stats.clj

```diff
@@ -298,8 +302,8 @@
 (defn- collection-metrics
   "Get metrics on Collection usage."
   []
-  (let [collections (t2/select Collection)
-        cards       (t2/select [Card :collection_id])]
+  (let [collections (t2/select Collection :creator_id [:not= config/internal-mb-user-id])
```

Comment:
According to all the blood in the last CI run, `Collection` doesn't have such a column 😢 

I'm guessing you can't abuse `:personal_owner_id` without hiding it from other users, so I'm not sure what you can do here. 

One dubious option would be to remove collections which only contain sample cards 🥴

Reply:
Yeah, I've updated this. I had some local state tricking me into thinking there was a creator_id column. 

https://github.com/metabase/metabase/pull/41519/files#diff-96de37bff643ff1187fa863f5a14541a6c03a6c2c20deaa7e043a0fcf8f54a08R305